malcolm/modules/pmac/infos.py

```diff
@@ -109,3 +115,16 @@ def in_cts(self, position: float) -> int:
         """Return the position (in EGUs) translated to counts"""
         cts = int(round((position - self.offset) / self.resolution))
         return cts
+
+    def check_position_within_soft_limits(self, position: float) -> bool:
+        """Check a position (in EGUs) against the soft limits and return True/False"""
+        # Soft limits of 0.0, 0.0 on a motor record means we should ignore them
+        if math.isclose(
```

Comment:
Do we need to do isclose? I though 0.0 was always 0.0 in floating point land? Also, does 0.0, 0.0 disable limits in drive limits or user limits?

Reply:
0.0 does seem to be 0.0 in floating point land - I am guessing it's safe because there's no operation on the value before comparison.

I now use dial limits to determine whether the soft limits are enabled or not.